app/api/sul_bib/publications_api.rb

```diff
@@ -17,37 +16,18 @@ def publication_find(id)
         error!({ 'error' => 'No such publication', 'detail' => "You've requested a non-existant publication." }, 404)
       end
 
-      # Check for existing authors.
-      # @param authorship_list [Array<Hash>]
-      # @return [boolean] true if any of the authors exist.
-      def existing_authors?(authorship_list)
-        # At least one of the authors in the authorship array must exist.
-        return false if authorship_list.nil?
-        authorship_list.any? { |auth| Contribution.authorship_valid?(auth) }
-      end
-
```

Comment:
Appears to be so, i.e.
```
$ git grep 'existing_authors?'  # on master
app/api/sul_bib/publications_api.rb:25:      def existing_authors?(authorship_list)
```
I suppose this prompts the question about what to do with it, the options are to remove it (as here) or find somewhere that it was supposed to be called.  Perhaps this was replaced by `Contribution.authorship_valid?` calls without removing the method (need to look at a git history or something to find the answer).  OK with this, I guess.